components/squeezelite/display.c

```diff
@@ -1202,7 +1255,7 @@ static void displayer_task(void *args) {
 		// update visu if active
 		if (visu.mode && visu.wake <= 0) {
 			visu_update();
-			visu.wake = 100;
+			visu.wake = 65; // decreaced refresh for ledvu.  possibly use visu.speed
```

Comment:
That bothers me a bit as we are very very short on CPU in some configuration and, regardless of priorities, running that too often is not great. Priorities do not solve the problem perfectly, I don't think I've left priority inversion but there will be some added scheduling time + more blocking where semaphore are used

Reply:
Put it back to 100 for now, but may want to look at making this configurable later